src/firestore/README.md

```diff
@@ -29,6 +29,10 @@ An index configuration file defines one object containing an <code>indexes</code
   ]
 }
 ```
+## Fetching your current configuration from the console
+
+Run `firebase firestore:indexes` to get a JSON list of all deployed indexes from the currently selected project. You can copy them into your 
```

Comment:
Nits: one wording, one hyphenation, "...of all deployed indexes for the currently-selected project...."